distributed/tests/test_client.py

```diff
@@ -6251,3 +6251,32 @@ async def test_get_task_metadata_multiple(c, s, a, b):
     assert len(metadata2) == 1
     assert list(metadata2.keys()) == [f2.key]
     assert metadata2[f2.key] == s.tasks.get(f2.key).metadata
+
+
+@gen_cluster(client=True)
+async def test_log_event(c, s, a, b):
+
+    # Log an event from inside a task
+    def foo():
+        get_worker().log_event("topic1", {"foo": "bar"})
+
+    assert not await c.get_events("topic1")
+    await c.submit(foo)
+    events = await c.get_events("topic1")
+    assert len(events) == 1
+    assert events[0]["foo"] == "bar"
```

Comment:
Why not something like this?

```suggestion
    assert events = ({"foo": "bar"},) 
```

Are we trying to support list/tuple ambiguity?  If so, I recommend calling `list(events)` or `tuple(events)` explicitly.

Reply:
There's an extra`"time"` timestamp key in each event which makes hard-coding the expected event tuple slightly more tricky. We could pop the `"time"` key off each event and then `assert` the observed tuple is as expected &mdash; which would be totally fine. I'm also okay with the current style here